spec/queries/popular_onet_codes_query_spec.rb

```diff
@@ -2,9 +2,10 @@
 
 RSpec.describe PopularOnetCodesQuery do
   it "returns the onet codes for popular occupation standards" do
-    create(:occupation_standard, title: "Medical Assistant", onet_code: "11")
-    create_list(:occupation_standard, 2, title: "Pipe Fitter", onet_code: "22")
-    create_list(:occupation_standard, 3, title: "Nurse", onet_code: "33")
+    create(:occupation_standard, :with_work_processes, title: "Medical Assistant", onet_code: "11")
+    create_list(:occupation_standard, 2, title: "HR Specialist", onet_code: "12")
```

Comment:
One minor request, just in an effort to create as few DB records as possible in tests:

Could you just create one record instead of 2, which should be sufficient as we just want to test that onet_code "12" doesn't show up in the list of our top 4:

```suggestion
    create(:occupation_standard, title: "HR Specialist", onet_code: "12")
```

And then to ensure that our test is always testing that we would like at most 4 onet codes returned, can you add:
```
stub_const("PopularOnetCodesQuery:LIMIT", 4) 
```
so that if this constant is ever changed to less than 4, we still want our test to work as if it had a limit of 4. I.e. without stubbing the constant, if LIMIT is changed to 3, our test would still pass, but it would no longer be testing that the "12" is not getting returned due to not having work processes, it would end up not getting returned because we were just requesting a limit of 3. But if we fix the LIMIT to 4 in our test regardless, then we can be sure that we are testing the correct thing.